exporter/datadogexporter/go.mod

```diff
@@ -158,7 +158,7 @@ require (
 	github.com/dennwc/varint v1.0.0 // indirect
 	github.com/digitalocean/godo v1.118.0 // indirect
 	github.com/distribution/reference v0.6.0 // indirect
-	github.com/docker/docker v27.0.3+incompatible // indirect
+	github.com/docker/docker v27.1.1+incompatible // indirect
```

Comment:
This is blocked, see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33757